integration-tests/bats/misc.bats

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+# Used for misc. tests that don't really fix anywhere else.
```

Comment:
No misc files in bats, these are already kind of a mess

A good place for this might be status.bats